core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersionResolver.scala

```diff
@@ -5,10 +5,15 @@ import java.nio.file.Path
 import com.google.gson.JsonParser
 
 import org.virtuslab.ideprobe.Extensions._
+import org.virtuslab.ideprobe.OS
 
 object IntelliJVersionResolver {
   def version(intelliJPath: Path): IntelliJVersion = {
-    val productInfo = intelliJPath.resolve("product-info.json").content()
+    val productInfo =
+      if (OS.Current == OS.Mac && intelliJPath.name == "Contents")
+        intelliJPath.resolve("Resources").resolve("product-info.json").content()
```

Comment:
Wow the amount of differences b/w Mac OS and Linux distributions of IntelliJs is surprising...

Reply:
yeah... Can't wait for adding Windows support here ^^

Reply:
No way :trollface: